### PR TITLE
update for 0.11

### PR DESCRIPTION
**Pull request by justinwoo**
Depends on https://github.com/purescript-contrib/purescript-dom/pull/89

### CODE REVIEW

**Comment by justinwoo** [comment]
updated and builds correctly 🍡

File: src/DOM/Classy/ParentNode.purs [hunk 1]
```diff
@@ -38,12 +34,12 @@ childElementCount = PN.childElementCount <<< toParentNode
 
 -- | Finds the first child that is an element that matches the selector(s), or
 -- | null if no such element exists.
-querySelector :: forall n eff. IsParentNode n => String -> n -> Eff (dom :: DOM | eff) (Nullable N.Element)
-querySelector selector = PN.querySelector selector <<< toParentNode
+querySelector :: forall n eff. IsParentNode n => String -> n -> Eff (dom :: DOM | eff) (Maybe N.Element)
```
Comment (garyb) [inline src/DOM/Classy/ParentNode.purs]: Can we update the argument to use `QuerySelector` rather than `String` here (and below)?

**Review by garyb — COMMENTED** [review]
All looks great apart from auto-wrapping QS!

File: src/DOM/Classy/ParentNode.purs (hunk 1 above)
Comment (justinwoo) [reply src/DOM/Classy/ParentNode.purs]: ah, yeah, might as well. pushed it up

File: src/DOM/Classy/ParentNode.purs [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
 import DOM.HTML.Types as H
 import DOM.Node.ParentNode as PN
 import DOM.Node.Types as N
-
 import Unsafe.Coerce as U
+import Control.Monad.Eff (Eff)
+import DOM (DOM)
+import DOM.Classy.Util (fromAny)
+import DOM.Node.ParentNode (QuerySelector)
+import Data.Maybe (Maybe)
```
Comment (garyb) [inline src/DOM/Classy/ParentNode.purs]: Sorry, last one, can we un-psc-ide the import ordering? :wink:

Comment (justinwoo) [reply src/DOM/Classy/ParentNode.purs]: hah, all right, pushed it up. i forgot letting psc-ide import stuff jumbles up the imports.

**Comment by justinwoo** [comment]
also got rid of the single import for queryselector

**Comment by garyb** [comment]
Thanks!

**Pull request merged**